### ISSUE TITLE
The given alias '`f`' is not part of any FROM or JOIN clause table

### ISSUE THREAD

**Issue by simonspa** [nextcloud/bookmarks#1061]
**Describe the bug**
Since the update to 3.1.0 I see occational errors in the Nextcloud log (see below), it looks like they appear during syncing with floccus. THis happens mostly for one of my users and I have not yet been able to reproduce this myself, hence the brief report. :/

**Server (please complete the following information):**
 - OS: Debian 10
 - HTTP server: nginx 1.14
 - Database: mysql 10.3.22
 - PHP version: 7.3.14
 - Nextcloud version: 18.0.4
 - Bookmarks app version: 3.1.0

**Nextcloud log (nextcloud/data/nextcloud.log)**
```
[index] Error: Doctrine\DBAL\Query\QueryException: The given alias '`f`' is not part of any FROM or JOIN clause table. The currently registered aliases are: `s`. at <<closure>>

 0. /var/www/nextcloud/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/Query/QueryBuilder.php line 1158
    Doctrine\DBAL\Query\QueryException::unknownAlias("`f`", ["`s`"])
 1. /var/www/nextcloud/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/Query/QueryBuilder.php line 1144
    Doctrine\DBAL\Query\QueryBuilder->verifyAllAliasesAreKnown({`s`: true})
 2. /var/www/nextcloud/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/Query/QueryBuilder.php line 1104
    Doctrine\DBAL\Query\QueryBuilder->getFromClauses()
 3. /var/www/nextcloud/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/Query/QueryBuilder.php line 238
    Doctrine\DBAL\Query\QueryBuilder->getSQLForSelect()
 4. /var/www/nextcloud/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/Query/QueryBuilder.php line 200
    Doctrine\DBAL\Query\QueryBuilder->getSQL()
 5. /var/www/nextcloud/lib/private/DB/QueryBuilder/QueryBuilder.php line 215
    Doctrine\DBAL\Query\QueryBuilder->execute()
 6. /var/www/nextcloud/apps/bookmarks/lib/Db/TreeMapper.php line 669
    OC\DB\QueryBuilder\QueryBuilder->execute()
 7. /var/www/nextcloud/apps/bookmarks/lib/Controller/FoldersController.php line 377
    OCA\Bookmarks\Db\TreeMapper->getChildren(31, 1)
 8. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 170
    OCA\Bookmarks\Controller\FoldersController->getFolderChildren(31, 1)
 9. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 99
    OC\AppFramework\Http\Dispatcher->executeController(OCA\Bookmarks\Co ... {}, "getFolderChildren")
10. /var/www/nextcloud/lib/private/AppFramework/App.php line 125
    OC\AppFramework\Http\Dispatcher->dispatch(OCA\Bookmarks\Co ... {}, "getFolderChildren")
11. /var/www/nextcloud/lib/private/AppFramework/Routing/RouteActionHandler.php line 47
    OC\AppFramework\App::main("OCA\\Bookmarks\ ... r", "getFolderChildren", OC\AppFramework\ ... {}, {folderId: "-1", ... "})
12. <<closure>>
    OC\AppFramework\Routing\RouteActionHandler->__invoke({folderId: "-1", ... "})
13. /var/www/nextcloud/lib/private/Route/Router.php line 299
    call_user_func(OC\AppFramework\ ... {}, {folderId: "-1", ... "})
14. /var/www/nextcloud/lib/base.php line 1008
    OC\Route\Router->match("/apps/bookmarks ... n")
15. /var/www/nextcloud/index.php line 38
    OC::handleRequest()

GET /index.php/apps/bookmarks/public/rest/v2/folder/-1/children?layers=1
from 92.211.208.180 by <USER> at 2020-06-03T14:29:14+02:00
```

**Comment by simonspa** [comment]
I do actually see that floccus fails to sync with the error `this.children is undefined` but there is no Nextcloud log entry for this - unrelated?

**Comment by marcelklehr** [comment]
It's related. It's a bug. It's fixed in v3.1.1 :)

**Comment by simonspa** [comment]
You are the best! Thank you!
